packages/components/src/components/ScalarMarkdown/ScalarMarkdown.vue

```diff
@@ -136,7 +137,7 @@ const html = computed(() => {
   .markdown p {
     color: inherit;
     line-height: 22px;
-    display: block;
+    display: inline;
```

Comment:
`<p>` should probably still be `block` by default, maybe we should just target `li > p`?

I feel like in fact our parser shouldn't be inserting `<p>` tags in our `<li>`s but if it's going to we probs only want to target those.

On another note does anyone know where this styling comes from headings in `<li>`'s? It seems to me we should just be parsing it to an inline `<strong>` tag

Reply:
Okay so I put it in a gist to see how Github parsed it and they _do_ seem to insert a `<p>` tag. 

https://gist.github.com/hwkr/8652ec2a8b15a1b8478034a8262c530c

They keep their display as `block` though so let's maybe just look at how they're doing it and try to match that?

<img width="2788" height="1968" alt="Google Chrome-2025-07-24-17-18-06@2x" src="https://github.com/user-attachments/assets/d34ff21d-b8a3-49d0-87fc-0577818d1397" />

Reply:
agree, `p { display: inline }` seems super dangerous, let’s avoid this